langlib/lang.typedesc/src/main/ballerina/typedesc_lib.bal

```diff
@@ -40,6 +40,18 @@ public type TypeId readonly & record {|
 |};
 
 # Returns the type-ids induced by a typedesc value.
+# 
+# ```ballerina
+# type Error distinct error;
+# type SampleError distinct (Error & error<record {string msg;}>);
```

Comment:
```suggestion
# type Error distinct error;

# type SampleError distinct (Error & error<record {string msg;}>);
```

Reply:
Fixed in https://github.com/ballerina-platform/ballerina-lang/pull/38983/commits/c857d393985333776f00303de8af5ae5204fa32b